### DISCUSSION TITLE
Guidance related to Gsoc Second issue.

### DISCUSSION THREAD

**Discussion by Rahulkhinchi03** [checkstyle/checkstyle#10032] [General]
Can anyone help me with understanding the gsoc second issue as I am Beginner? 

update SuppressWarningsCheckTest to use unique Input file in each test method #9521

https://github.com/checkstyle/checkstyle/issues/9521

**Comment by romani** [comment]
Take a look at example https://github.com/checkstyle/checkstyle/commit/c7e81c1dff601912c54d7cc828468048385fc71f

All input files if any, should be used only by one test method in whole project.

https://github.com/checkstyle/checkstyle/blob/de58a23d408d79609af3e84b2d14d69900a0a8e4/src/test/java/com/puppycrawl/tools/checkstyle/checks/annotation/SuppressWarningsCheckTest.java#L56

There should be no other test method that use this input file.

**Reply by Rahulkhinchi03** [reply]
Thank you for your response!
